user/disable-job-logs.md

```diff
@@ -0,0 +1,36 @@
+Similarly, this setting allows you to restrict access to build job logs for any user without write/push access rights to the repository. Limit job log visibility to only those that needed it. Enable this setting and ensure job logs are only available to users with respective read or write access to the individual repository.
+
+The following are the available configurations:
+* If this setting is ON, it allows access to build job logs only for users with write/push permissions to this repository. Limits access to build job logs via UI and API. 
+* If this setting is OFF, users with read access to the repository can access the build job logs. 
+
+Please note that the '*Limiting access to build job logs*' repository setting applies only to users with ‘write/push’ permissions.
+
+### Disable logs Options 
+
+|                                                                                                | Public repository                                                                                                                                                                                                                                                                                                                       | Private repository                                                                                |
+|------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| **Access to logs only for write/push users OFF, Access to old logs OFF (default settings)**        | Users with **read access** to the repository can see build job logs for public repo not older than the last X days.                                                                                                                                                                                                                             | Users with **read access** to the repository can see the build job logs not older than the last X days.   |
+| **Access to logs only for write/push users ON, Access to old logs OFF**                            | Users with **write access** to the repository will see the build job logs not older than the last X days.                                                                                                                                                                                                                                       | Users with **write access** to the repository will see the build job logs not older than the last X days. |
```

Comment:
Instead of "X days" should be "365 days"

Reply:
This was updated! Thanks for the feedback